lib/CL/clCreateProgramWithIL.c

```diff
@@ -28,6 +28,73 @@
 #include "pocl_util.h"
 #include "pocl_shared.h"
 
+static int
+get_program_spec_constants (cl_program program, char *program_bc_spirv)
+{
+  char *args[] = { LLVM_SPIRV, "--spec-const-info", program_bc_spirv, NULL };
+  char captured_output[8192];
+  size_t captured_bytes = 8192;
```

Comment:
Please use constant variables or defines for magic numbers.

Reply:
fixed